synapse/res/templates/sso_auth_confirm.html

```diff
@@ -0,0 +1,15 @@
+<html>
+<head>
+    <title>Authentication</title>
+</head>
+    <body>
+        <div>
+            <p>
+                A client is trying to remove a device/add an email address/take over
```

Comment:
assuming that you know this text needs changing!

Reply:
Yes, but thank you for marking it! 👍

Reply:
Any thoughts on how customizable we want this to be? Do we want the "reason" to be templated or static?

Reply:
I don't think I understand the question. The "reason" is defined by the operation the client is requesting, so it needs to be substituted somehow?

It might be worth seeking some feedback in the "Write Club" room (#write-club-internal:matrix.org) for suggestions on what this page ought to look like.